go/filemanager/pfsmodules/status.go

```diff
@@ -0,0 +1,23 @@
+package pfsmodules
+
+// TODO
+// need a custom error type?
+
+const (
+	StatusFileNotFound          = "no such file or directory"
```

Comment:
I am still not convinced that we need these string constants, and these string constant need to be visible outside of the package.
Can you take a look at : https://github.com/PaddlePaddle/cloud/pull/112#discussion_r119712517